examples/mnist/quantized.cpp

```diff
@@ -45,6 +45,7 @@ void construct_net(network<sequential> &nn) {
      << quantized_convolutional_layer<tan_h>(
           5, 5, 5, 16, 120, padding::valid, true, 1, 1,
           backend_type)  // C5, 16@5x5-in, 120@1x1-out
+     << tanh_layer(120)
```

Comment:
That sums up with the previous comment. I added at one place and forgot at another during rebasing and removing merge conflict headers..